### PR TITLE
Filename conventions

### PR DESCRIPTION
**Pull request by kodster28**
Related to #15223

### CODE REVIEW

File: content/style-guide/documentation-content-strategy/filenames.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Filenames should:
+- Semantically communicate the purpose of the file
+- Be lowercased
+- Use dashes dashes between words
+
+```txt
+---
+header: Acceptable file names
+---
+/fundamentals/concepts/what-is-cloudflare/
```
Comment (KianNH) [inline content/style-guide/documentation-content-strategy/filenames.md]: Should we use `/fundamentals/concepts/what-is-cloudflare/index.md` here, or call-out separately that `/foo/index.md` maps to what is shown on `/foo/`?

Comment (kodster28) [reply content/style-guide/documentation-content-strategy/filenames.md]: Ugh, @pedrosousa and I were just chatting about this.
(you moving up the corrections leaderboard)

Comment (kodster28) [reply content/style-guide/documentation-content-strategy/filenames.md]: Should be addressed by the latest few pushes

File: content/style-guide/documentation-content-strategy/file-conventions.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+```
+
+```txt
+---
+header: Unacceptable file names
+---
+/fundamentals/concepts/What is Cloudflare.md
+/fundamentals/concepts/What-is-Cloudflare.md
+/fundamentals/concepts/what-is-cloudflare/index.md
```
Comment (pedrosousa) [inline content/style-guide/documentation-content-strategy/file-conventions.md:35]: Nit: There's one single use case for this, so I wouldn't consider it an error in a future linter: when you plan to expand a section very soon but do not want an expansion arrow in the sidebar (caused by using `_index.md`).

File: content/style-guide/documentation-content-strategy/file-conventions.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+```
+
+This convention ensures that [Hugo](https://gohugo.io/) - our static site generator - treats the content as a [section](https://gohugo.io/content-management/sections/). The content at `/fundamentals/concepts/_index.md` will also be rendered at `https://developers.cloudflare.com/fundamentals/concepts/`.
+
+## Content files
+
+Add regular content files to the `/content/{product_folder}/` directory.
+
+```txt
+/fundamentals/concepts/what-is-cloudflare/
```
Comment (pedrosousa) [inline content/style-guide/documentation-content-strategy/file-conventions.md]:
```suggestion
/content/fundamentals/concepts/what-is-cloudflare.md
```
This code block contains an example public URL -- should it contain an example file path instead?

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**